test/acceptance/specs/CookieSettingsPageSpec.scala

```diff
@@ -115,11 +116,6 @@ class CookieSettingsPageSpec extends BaseAcceptanceSpec {
 
     And("the dataLayer contains the 'trackingConsentSettingsAccepted' event")
     settingsAllowedGtmEvent should not be null
-
```

Comment:
this is outside the scope of this test of GTM events - there's a specific test for setting the cookie (the next one)